### ISSUE TITLE
Cursor pointer image wrong behavior

### ISSUE THREAD

**Issue by DiedByDisgust** [hyprwm/Hyprland#1097]
after last system upgrade(Arch) mouse pointer keeps getting locked
examples:
![image](https://user-images.githubusercontent.com/25803026/203732476-0bedf056-c91f-48c9-b266-101914bfd09c.png)
*a grabbing hand although isn't anything grabbed

![image](https://user-images.githubusercontent.com/25803026/203733126-c2068c8b-e9dd-4aae-be1b-b493dc3c7c62.png)

the lock is relative to window, in each different window keeps a particular mode

i can't tell specific instructions for you to reproduce, i just "pacman -Syu" my system

**Comment by vaxerski** [comment]
the hand should be enabled when dragging a window with the mouse.

**Comment by KyrillGobber** [comment]
Same here. In librewolf (and probably also firefox) my cursor sometimes completely disappears.

**Comment by vaxerski** [comment]
"probably also" I use firefox exclusively and have no issues.

**Comment by DiedByDisgust** [comment]
Yes. But it keep that pointer image. Or if it's just an "arrow" dont turn onto "finger"👆 during links hovering, or so

**Comment by DiedByDisgust** [comment]
> Same here. In librewolf (and probably also firefox) my cursor sometimes completely disappears.

Also this.

**Comment by vaxerski** [comment]
please find a method to reproduce. I have *not once* had that happen. Without a reproducible case, it's hard to fix.

**Comment by DiedByDisgust** [comment]
> please find a method to reproduce. I have _not once_ had that happen. Without a reproducible case, it's hard to fix.

Fixed by downgrade gtk3 package.
```

[2022-11-24T16:37:16-0500] [ALPM] upgraded gtk3 (1:3.24.34+r156+g812b3930d0-1 -> 1:3.24.35-1)
[2022-11-24T16:37:16-0500] [ALPM] upgraded lib32-gtk3 (3.24.34+r156+g812b3930d0-1 -> 1:3.24.35-1)
```

:+1:  3.24.34+r156+g812b3930d0-1 
:-1:   3.24.35-1

but now what? should i take this to gtk team?

**Comment by vaxerski** [comment]
probably.

**Comment by KyrillGobber** [comment]
> "probably also" I use firefox exclusively and have no issues.

Sorry for that. You're right, I should be more accurate.
I installed firefox and I have the same problem there.

Now I downgraded the gtk package as @DiedByDisgust said and the issue is gone. But on my laptop I don't have the issue no matter what gtk version, and no way to reproduce.

What info can I provide to help solving this?

**Comment by vaxerski** [comment]
Pretty much none, to me. Seems like a gtk issue and should be reported there. IIRC, from one of the logs sent by someone, "left_ptr" fails to load which would definitely be a gtk bug

**Comment by vaxerski** [comment]
https://gitlab.gnome.org/GNOME/gtk/-/issues/5365

**Comment by vaxerski** [comment]
https://bugs.archlinux.org/task/76651

**Comment by vaxerski** [comment]
The way to repro this is to upgrade and have your cursor theme in `~` instead of `/etc/`

**Comment by KyrillGobber** [comment]
Problem solved with latest gtk3 1:3.24.35-2
